### ISSUE TITLE
Getting error while creating Responsible Ai dashboard 

### ISSUE THREAD

**Issue by yogidosalwar** [Azure/mlops-v2#67]
I am trying to use different algorithms for training. Model training and model registration part is successfully completed, but RAI insights dashboard constructor is failed. I am using the xgboost model for training.

```
Traceback (most recent call last):
  File "create_rai_insights.py", line 184, in <module>
    main(args)
  File "create_rai_insights.py", line 128, in main
    model_estimator = load_mlflow_model(my_run.experiment.workspace, model_id=model_id)
  File "/mnt/azureml/cr/j/exe/wd/rai_component_utilities.py", line 79, in load_mlflow_model
    return mlflow.pyfunc.load_model(model_uri)._model_impl
  File "/azureml-envs/responsibleai-0.18/lib/python3.8/site-packages/mlflow/pyfunc/__init__.py", line 484, in load_model
    model_impl = importlib.import_module(conf[MAIN])._load_pyfunc(data_path)
  File "/azureml-envs/responsibleai-0.18/lib/python3.8/site-packages/mlflow/sklearn/__init__.py", line 494, in _load_pyfunc
    return _load_model_from_local_file(path=path, serialization_format=serialization_format)
  File "/azureml-envs/responsibleai-0.18/lib/python3.8/site-packages/mlflow/sklearn/__init__.py", line 452, in _load_model_from_local_file
    return cloudpickle.load(f)
ModuleNotFoundError: No module named 'xgboost'
```

Thank You

**Comment by setuc** [comment]
Can you share which environment are you using? It's possible that the environment doesn't contain xgbosst. Consider adding that package as part of conda.yaml
@yogidosalwar

**Comment by yogidosalwar** [comment]
@setuc  Here is my conda.yaml
<img width="765" alt="artifact" src="https://user-images.githubusercontent.com/49486755/199459983-6b06177c-5879-4366-b963-d57e29582483.png">

**Comment by setuc** [comment]
Do you have your pipeline? Or did you just replace the model with XGBoost?

**Comment by yogidosalwar** [comment]
@setuc just i replace the model with XGBoost and added model dependencies  in requirment.txt , train-conda.yml and train-monitor-conda.yml.

**Comment by yogidosalwar** [comment]
@setuc  here is screen shot of DevOps
<img width="758" alt="devops" src="https://user-images.githubusercontent.com/49486755/199462712-9468d676-4d4f-4248-adfb-4fdc5e6a09dc.png">

Thank you.

**Comment by setuc** [comment]
Ahh. Got it. The responsbileAi modules user the environment that's different than the base container. It refers to the container from the Rai-vnext-Preview code. You can check the Rai component yaml file to see how the environment file is being built.

**Comment by yogidosalwar** [comment]
Ok. I will check.
Thank you.

**Comment by yogidosalwar** [comment]
@setuc I went through the Rai-component.yml file and I made changes in requirment.txt which is part of Rai-vnext-Preview.
But still, getting error in the Run Azure ML pipeline. If you have any input, please let me know.

<img width="585" alt="Devops1" src="https://user-images.githubusercontent.com/49486755/199500740-b7fe0c3c-41b5-4af0-9b3e-5ea0ee279505.png">

Thank you.

**Comment by cindyweng** [comment]
Can you see what's going on in the AML pipeline? Via the studio

**Comment by yogidosalwar** [comment]
@cindyweng @setuc  I have RAI-vNext-Preview repo and MLOps-Test folder on my github. I made some changes as per my requirement in deploy-model-training-pipeline. yml, prep. py etc. and as well as I did changes in create_rai_insights.py. While running deploy-model-training-pipeline getting error while creating RAI Insights Dashboard Constructor. I went through the azure machine learning studio jobs and observe the code, but my changes are not showing there.

i did changes here 
parser.add_argument("--maximum_rows_for_test_dataset", type=int, default=5000)
to 
parser.add_argument("--maximum_rows_for_test_dataset", type=int, default=10000)
<img width="739" alt="responsibleai" src="https://user-images.githubusercontent.com/49486755/206172209-bfb98e39-2079-4041-99a0-20f1b12e805c.png">

below changes i did in deploy-model-training-pipeline.yml 
```
    - repository: RAI-vNext-Preview  # Template Repo  #rai-vnext-preview
      name: yogidosalwar/RAI-vNext-Preview # need to change org name from "Azure" to your own org #rai-vnext-preview
      endpoint: github-connection # need to set up and hardcode
      type: github
      ref: main  
```
Thank you
